Code/Source/Spawner/ROS2SpawnerComponent.cpp

```diff
@@ -213,10 +214,7 @@ namespace ROS2
             pose.position.x = info.pose.GetTranslation().GetX();
             pose.position.y = info.pose.GetTranslation().GetY();
             pose.position.z = info.pose.GetTranslation().GetZ();
-            pose.orientation.x = info.pose.GetRotation().GetX();
-            pose.orientation.y = info.pose.GetRotation().GetY();
-            pose.orientation.z = info.pose.GetRotation().GetZ();
-            pose.orientation.w = info.pose.GetRotation().GetW();
+            pose.orientation = ROS2Conversions::ToROS2Quaternion(info.pose.GetRotation());
```

Comment:
It would be nice to have `ToROS2Pose` function. I might not remember correctly but conversion from AZ::Vector3 to ROS vector3 is also available so it could be use there as well